### DISCUSSION TITLE
Official "publish" feature?

### DISCUSSION THREAD

**Discussion by pckim93** [pdm-project/pdm#1070] [Ideas]
How would I "publish" a package to pypi (the official pypi or other clones, including AWS CodeArtifact)?
I guess poetry has this feature although I haven't really used it.  I know there's a pdm-publish project, but it looks like someone's personal project and the project itself was using poetry instead of pdm.

If there's no such capability at the moment, is there a plan to add one? (Of course, I can try to contribute, but wanted to know the plans first.) Thanks!

**Comment by Hnasar** [comment]
Since PDM is a valid PEP 517 backend, you can just use the standard `build` package to build a wheel. Then you can use `twine` to upload it to a package registry like pypi.

https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives
